.changeset/new-plants-sort.md

```diff
@@ -0,0 +1,8 @@
+---
+'@backstage/plugin-kubernetes-react': patch
+'@backstage/plugin-kubernetes': patch
+---
+
+Add `authuser` search parameter to GKE cluster link formatter in k8s plugin
+
+Thanks to this, people with multiple simultaneously logged-in accounts to be automatically picked.
```

Comment:
Checking my understanding here:
Suppose you're signed in with the Google auth provider in Backstage. This PR ensures that, when you click the dashboard link to a Kubernetes object in a GKE cluster, you will be viewing that dashboard with the same identity as the one currently logged in to Backstage? When you say "multiple simultaneously logged-in accounts", does that mean multiple accounts on the Google Cloud side where the dashboards are?

Reply:
Yes, it means Google side.
GCP console has annoying behaviour of redirecting to different places when you try to access resources you do not own.
And in my example, I have a different google account for email, and different specific one for google cloud.

Reply:
I have amended the changeset. Hopefully it's more clear now.